### DISCUSSION TITLE
Evaluate FEFunction on arbitrary point of the domain

### DISCUSSION THREAD

**Discussion by IvanBioli** [gridap/Gridap.jl#1044] [Q&A] [answered]
Hi, 

Thank you for your incredible effort in developing `Gridap.jl`! 
I was wandering: is there any way to evaluate a `Gridap.FEFunction` at an arbitrary point of the domain?

Thanks!

**Comment by amartinhuertas** [comment, marked as answer]
Yes. If `uh` is a Gridap's `FEFunction`, and `x` is an arbitrary point in the domain (of type `Point{D}`, where D is the number of space dimensions), then you can write `uh(x)` to achieve this.

**Comment by JordiManyer** [comment]
Just to add to what @amartinhuertas said: Note that this is incredibly inefficient since it will have to do a global tree-search to locate which cell contains which point. Also, this search is relies on inverse-cellmaps. So we do not recommend this unless it is for debugging/plotting purposes.

**Reply by IvanBioli** [reply]
Thanks a lot to both of you for the detailed answer. I expected a very inefficient function as one needs to find the cell containing the point, but I just need it as a sanity check.
